.github/workflows/e2e-tests.yml

```diff
@@ -48,6 +48,12 @@ jobs:
           reporter: jest-junit
           output-to: step-summary
 
+      - name: output test screenshots from the final state of tests
+        uses: actions/upload-artifact@v3
+        with:
```

Comment:
You need if always otherwise the screenshots won't be uploaded if the tests failed